tests/util/ast.js

```diff
@@ -96,8 +96,8 @@ describe('ast', () => {
 
       assert.strictEqual(enterCalls.length, 6);
 
-      enterCalls.forEach((node, idx) => {
-        assert.strictEqual(node.lastArg.argument.value, idx);
+      enterCalls.forEach((call, idx) => {
+        assert.strictEqual(call.args[0].value, idx);
```

Comment:
why the change from last arg to first arg?

Reply:
Previously the called function has only one parameter: `(node) => void`, I refactored that function to `(node, fn) => void`. The last arg changed to a different thing.